_pages/en_US/installing-boot9strap-(safecerthax).txt

```diff
@@ -0,0 +1,84 @@
+{: .notice--info}
+
+### Compatibility Notes
+
+safecerthax is compatible with all Old 3DS and Old 2DS devices in all regions on system versions 1.0.0 through 11.14.0. 
+
+The instructions provided only apply to the New 3DS, New 3DS XL, and New 2DS XL. Please ensure that the device you are modding is a New 3DS, New 3DS XL, or New 2DS XL before continuing.
+{: .notice--warning}
+
+### What You Need
+
+* The latest release of [SafeB9SInstaller](https://github.com/d0k3/SafeB9SInstaller/releases/download/v0.0.7/SafeB9SInstaller-20170605-122940.zip) (direct download)
+* The latest release of [boot9strap](https://github.com/SciresM/boot9strap/releases/download/1.4/boot9strap-1.4.zip) (direct download)
+* The latest release of [Luma3DS](https://github.com/LumaTeam/Luma3DS/releases/latest) (the Luma3DS .zip file)
+
+#### Section I - Hardware Button Check
+
+{% include_relative include/safemodecheck.txt %}
+
+If you do not boot into Safe Mode, the system update menu after multiple attempts, you cannot follow this method. If this is the case, use [Seedminer](seedminer) instead.
+{: .notice--danger}
+
+#### Section II - Prep Work
+
+In this section, you will copy the files needed to trigger the safecerthax exploit.
+
+1. Insert your SD card into your computer
+1. Copy `boot.firm` and `boot.3dsx` from the Luma3DS `.zip` to the root of your SD card
+    + The root of the SD card refers to the initial directory on your SD card where you can see the Nintendo 3DS folder, but are not inside of it
+1. Create a folder named `boot9strap` on the root of your SD card
+1. Copy `boot9strap.firm` and `boot9strap.firm.sha` from the boot9strap `.zip` to the `/boot9strap/` folder on your SD card
+1. Copy `SafeB9SInstaller.bin` from the SafeB9SInstaller `.zip` to the root of your SD card
+1. Reinsert your SD card into your device
+1. Power on your device
+
+#### Section III - safecerthax Proxy
+
+In this section, you will change your Internet connection settings to use a proxy network designed to exploit the System Update feature of your device.
+
+{% include_relative include/addproxy.txt %}
+1. Power off your device
+
+#### Section IV - safecerthax
+
+In this section, you will enter Safe Mode (a feature available on all 3DS family devices) where safecerthax will be triggered, which will launch you into the boot9strap (custom firmware) installer.
+
+1. With your device still powered off, hold the following buttons: (Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A), and while holding these buttons together, power on your device
+  + Keep holding the buttons until the device boots into Safe Mode (a "system update" menu)
+  + If you're unable to get into Safe Mode after multiple attempts, one of your buttons may be failing or broken. If this is the case, you will need to follow [an alternate exploit](seedminer). For assistance with this matter, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and ask, in English, for help.
```

Comment:
Inline troubleshooting should no longer be necessary here now that the check is made mandatory in Section I

Reply:
hold this change to another PR as USM has the same issue

Reply:
keeping this unresolved to keep track, no need to change in this pr